README.md

```diff
@@ -180,3 +180,19 @@ To set environment variables, you can place them as a key value pair into a `.st
 file inside your source code repository.
 
 Example: DATABASE_USER=sampleUser
+
+Hot deploy
+---------------------
+Hot deploy is not supported in this image. To support it, your application needs to be built with one of modules that reloads the server every time changes in source code are done inside the running container. Those modules are:
```

Comment:
yeah but it feels a little weird to say it's "built with supervisor", you're just installing supervisor (via npm install) as a separate action from building your app, in my mind.